.pre-commit-config.yaml

```diff
@@ -0,0 +1,18 @@
+repos:
+  - repo: local
+    hooks:
+      - id: black
+        name: black
+        language: system
+        entry: black
```

Comment:
Can we add the `--skip-string-normalization` option here?

Reply:
I like the `'` more than the `"` but I hate having to switch to what each project wants even more. If we skip string normalization we'll have a mix of `"` and `'` pretty soon.